lnrpc/rpc.proto

```diff
@@ -1862,6 +1862,12 @@ message OpenChannelRequest {
     transaction.
     */
     uint32 max_local_csv = 17;
+
+    /*
+    The explicit commitment type to use. Note this field will only be used if
+    the remote peer supports explicit channel negotiation.
```

Comment:
Still going through the diff so maybe this'll be answered by a later commit. But do we need a way to control what happens if the remote peer _does not_ support the commitment type? Or will the negotiation always fail if the desired type cannot be agreed upon?

Reply:
It's up to the remote peer to reject it if it doesn't support the extended commitment type. There's a requirement for them to echo the channel type back in `accept_channel`, which they could blindly do so. Negotiation would eventually fail though when they exchange signatures for their commitment transactions (assuming the commitment type changes the structure of commitment transactions).